simple_token_authentication.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "actionmailer", "5.0.0.beta3"
   s.add_dependency "actionpack", "5.0.0.beta3"
-  s.add_dependency "devise", "4.0.0.rc2"
+  s.add_dependency "devise", "~> 4.0"
```

Comment:
Hi @EppO,

Supporting Devise 4 doesn't mean dropping support for Devise 3 - see https://github.com/gonzalo-bulnes/simple_token_authentication/pull/246/commits/775275418e3e91fc8ec42d0befbf30601ad8547e